.github/workflows/build.yml

```diff
@@ -10,6 +10,12 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     steps:
+      # Windows throws false positives with linting because of CRLF
+      - name: Set git to use LF
+        run: |
+          git config --global core.autocrlf false
```

Comment:
For my understanding, not setting this would cause the `npm run lint` to fail below?

I'm also a bit confused on the right settings -- the [docs](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration) say that setting this to false will allow CRLF endings? Is that what we want for all platforms?

Reply:
Yes, the linting fails without `git config --global core.autocrlf false`, example: https://github.com/vcheung-stripe/vscode-stripe/runs/2603830207?check_suite_focus=true

My understanding is that git automatically converts LF to CRLF, and setting `core.autocrlf false` turns off the conversion.

Reply:
I see, thanks for explaining!